docs/project/overview.md

```diff
@@ -4,66 +4,78 @@
 
 ## Context
 
-In modern IT infrastructures, ensuring the reliability and availability of servers and services is crucial. The growing complexity of server environments and the increasing volume of data generated by these systems have created a pressing need for real-time monitoring and proactive anomaly detection.
 
-This project was initiated to address these challenges by creating a scalable real-time monitoring platform. The system will leverage advanced data analysis and machine learning techniques to monitor server health, detect performance anomalies, and alert system administrators to potential issues before they escalate into failures.
+**Educado** is a web and mobile application developed as part of the **Egalitarian Erasmus Initiative**, an effort within the **Egalitarian SDG Challenge** and supported by the **Erasmus+ Programme**. The project involves collaboration among students from:
 
-The project's aim is to reduce downtime, increase operational efficiency, and minimize the need for manual intervention in server management, ultimately improving the overall stability of IT systems.
+- **University of Brasília (Brazil)**  
+- **Aalborg University (Denmark)**  
+- **University of Minho (Portugal)**  
+- **Saxion University (Netherlands)**  
+
+Educado is designed to **empower waste pickers** by providing educational resources that improve their financial literacy, health, machine operation skills, and hygiene practices. Teachers can upload educational content through a web app, while waste pickers can access the classes through a mobile app. By equipping waste pickers with accessible learning tools, the project contributes to **Sustainable Development Goal (SDG) 4: Quality Education**.
 
 ---
 
 ### Objectives and Scope
 
 #### Objectives:
 
-* Develop a scalable and efficient real-time monitoring system capable of tracking server health across a wide range of metrics (e.g., CPU, memory, disk usage, network traffic).
-* Implement machine learning algorithms to analyze historical and real-time server data, detecting anomalies and predicting potential hardware failures.
-* Design an alerting system that notifies administrators of detected issues through various channels, including email, SMS, and integration with messaging platforms like Slack.
-* Create an intuitive dashboard to provide administrators with real-time insights into server status, trends, and health indicators, allowing for informed decision-making.
+* Develop a **web platform for teachers** to upload educational videos and materials.
+* Develop a **mobile application** that delivers **educational content** tailored for waste pickers.
+* Improve **financial literacy, health awareness, and professional skills** within the waste-picking community.
+* Provide **easy-to-understand learning materials** through multimedia content.
+* Promote **digital inclusion** by ensuring accessibility and usability of the application.
```

Comment:
Implement "digital accessibility guide" to ensure all HCI guidelines were followed